packages/core/core/src/factories/useCategoryFactory.ts

```diff
@@ -1,6 +1,6 @@
 import { UseCategory } from '../types';
-import { useSSR } from '../utils';
-import { ref, Ref, computed } from '@vue/composition-api';
+import { Ref, computed } from '@vue/composition-api';
+import { ssrRef } from 'nuxt-composition-api';
```

Comment:
this (and all others) has to be abstracted to work on non-nuxt environments as well.  We can also just use `if` to check if current env is nuxt or not and use `ssrRef` or `ref`